gatsby-config.js

```diff
@@ -473,5 +473,13 @@ module.exports = {
         allExtensions: true, // defaults to false
       },
     },
+    {
+      resolve: `gatsby-alias-imports`,
+      options: {
+        aliases: {
+          images: path.resolve(__dirname, 'src/images/'),
+        },
+      },
+    },
```

Comment:
This is only marginally less code than doing it ourselves and not pulling in another dependency. If you look at the 
 [source](https://github.com/luanbitar/gatsby-alias-imports/blob/main/gatsby-node.js) of this plugin, it's really only parsing the plugin config and outputting one line to the webpack config. I would recommend just following the docs for [webpack alias](https://github.com/luanbitar/gatsby-alias-imports/blob/main/gatsby-node.js) and put it directly into the config.

Reply:
@aswanson-nr Great callout - I'll make that change :)